package cms_8120;
option java_package = "com.hikvision.finance.fms.cms.hpp.type";
option java_outer_classname = "ModuleTypeProto";
// @模块类型定义
// 字典大类:4
enum MODULE_TYPE {
    MT_SERVER_ADJUSTTIME              = 20001;// 系统巡检服务
    MT_SERVER_MEDIAGUARD              = 20002;// 录像守卫服务
	MT_SERVER_TRANSPORT               = 20004;// 上传服务
	MT_CLIENT_CS					  = 20005;// CS客户端						  
}